recipes/glib/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# don't get locally installed pkg-config entries:
+export PKG_CONFIG_LIBDIR="${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig"
```

Comment:
I am 80% sure that it is needed since I have to do that in most of the other packages that use pkgconfig.